src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -530,55 +538,57 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
 
           {/* These controls are automatically hidden when the side sheet is open. */}
           <Box width="100px" ml={2} display={drawerIsOpen ? 'none' : 'block'}>
-            <Box
-              display="flex"
-              mb={2}
-              pb={2}
-              width="100%"
-              sx={{ borderBottom: `1px solid ${theme.palette.gray[600]}` }}
-            >
-              <Tooltip title="Previous Run" arrow>
-                <Button
-                  sx={{ fontSize: '28px', px: 0, flex: 1 }}
-                  variant="text"
-                  onClick={() => {
-                    // This might be confusing, but index 0 is the most recent run, so incrementing the index goes
-                    // to an *earlier* run.
-                    dispatch(selectResultIdx(selectedResultIdx + 1));
-                    navigate(
-                      `?workflowDagResultId=${
-                        workflow.dagResults[selectedResultIdx + 1].id
-                      }`
-                    );
-                  }}
-                  disabled={
-                    selectedResultIdx === workflow.dagResults.length - 1
-                  }
-                >
-                  <FontAwesomeIcon icon={faChevronLeft} />
-                </Button>
-              </Tooltip>
-
-              <Tooltip title="Next Run" arrow>
-                <Button
-                  sx={{ fontSize: '28px', px: 0, flex: 1 }}
-                  variant="text"
-                  onClick={() => {
-                    // This might be confusing, but index 0 is the most recent run, so decrementing the index goes
-                    // to a *newer* run.
-                    dispatch(selectResultIdx(selectedResultIdx - 1));
-                    navigate(
-                      `?workflowDagResultId=${
-                        workflow.dagResults[selectedResultIdx - 1].id
-                      }`
-                    );
-                  }}
-                  disabled={selectedResultIdx === 0}
-                >
-                  <FontAwesomeIcon icon={faChevronRight} />
-                </Button>
-              </Tooltip>
-            </Box>
+            {workflow.dagResults && workflow.dagResults.length > 1 && (
```

Comment:
only show 'prev version' / 'next version' buttons if there are more than one results for the workflow. The rest LOCs are just result of the tab.